test/e2e/framework/util.go

```diff
@@ -2066,16 +2066,29 @@ func AssertCleanup(ns string, selectors ...string) {
 	if ns != "" {
 		nsArg = fmt.Sprintf("--namespace=%s", ns)
 	}
+	var err error
+	for i := 1; i <= 3; i++ {
```

Comment:
Wouldn't be better to use `wait.ExponentialBackoff` instead?

Reply:
@soltysh I wasn't aware of the function `wait.ExponentialBackoff`. I took a look at it and it is a better approach. I will update the change. Thank you for your suggestion.